.github/workflows/unitest.yml

```
@@ -0,0 +1,43 @@
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Dependents
+        run: |
+          sudo apt-get install -y portaudio19-dev python3-pyaudio
+
+      - name: Create venv
+        run: python3 -m venv .venv
+
+      - name: Activate venv
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:2:11: Double quote to prevent globbing and word splitting

(shellcheck)